globus_sdk/globus.cfg

```diff
@@ -2,30 +2,36 @@
 
 [environment default]
 auth_service = https://auth.globus.org/
-transfer_service = https://transfer.api.globus.org/
 nexus_service = https://nexus.api.globusonline.org/
+transfer_service = https://transfer.api.globus.org/
+search_service = https://search.api.globus.org/
 
 [environment preview]
 auth_service = https://auth.preview.globus.org/
 nexus_service = https://nexus.api.preview.globus.org/
+search_service = https://search.api.preview.globus.org/
 transfer_service = https://transfer.api.preview.globus.org/
 
 [environment sandbox]
 auth_service = https://auth.sandbox.globuscs.info/
-nexus_service = https://graph.api.go.sandbox.globuscs.info/
+nexus_service = https://graph.api.sandbox.globuscs.info/
+search_service = https://search.api.sandbox.globuscs.infog/
```

Comment:
It looks like all these search URLs have a typo? ("g")

Also, I think we need to switch all these `graph` entries to `nexus`.